redwind/contexts.py

```diff
@@ -46,20 +46,20 @@ def extract_ogp_context(context,doc,url):
     ogp_url = soup.find('meta',{'property':'og:url'})
     ogp_content = soup.find('meta',{'property':'og:description'})
 
-    if ogp_title:
+    if ogp_title and not context.title:
         context.title = ogp_title['content']
 
-    if ogp_image:
+    if ogp_image and not context.author_image:
         context.author_image = ogp_image['content']
 
-    if ogp_site:
+    if ogp_site and not context.author_name:
         context.author_name = ogp_site['content']
 
-    if ogp_url:
+    if ogp_url and not context.permalink:
         context.url = ogp_url['content']
```

Comment:
I think we only want to fill in the permalink here, the URL should come from the user-entered URL (and should already be filled in)